### PR TITLE
Use host network stack when using docker-compose.

### PR DESCRIPTION
**Pull request by Rombur**

### CODE REVIEW

**Comment by Rombur** [comment]
@sslattery this should fix the problem you have with docker

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -22,3 +22,5 @@ services:
     # for reference.
     security_opt:
       - seccomp:unconfined
+    # Use host network stack due to some problems with ORNL firewall on Ubuntu
```
Comment (dalg24) [inline docker/docker-compose.yml]: Is that really a firewall issue?  Has anybody tried outside the lab?

Comment (Rombur) [inline docker/docker-compose.yml]: When I use docker at home I don't need `-net=host` to connect to internet.

Comment (dalg24) [inline docker/docker-compose.yml]: OK.  Maybe expand a bit your comment to address @aprokop review.

**Comment by Rombur** [comment]
@aprokop without this, we can't get our ubuntu machines (Stuart and me) to connect to internet. So instead of the container using it's own network stack, it use the stack of the host machine.

**Comment by aprokop** [comment]
> instead of the container using it's own network stack, it use the stack of the host machine.

Thanks! Could you please add this comment to the patch? That would be helpful in the future.

And I still can't quite understand why it solves the problem.

**Comment by Rombur** [comment]
@aprokop Honestly, I am not sure either but according to google when you have a problem with your firewall that's what you need to do ;-)

**Comment by Rombur** [comment]
I have improved the comment.

**Pull request merged**